Makefile

```diff
@@ -81,10 +81,11 @@ propose-jwt-demo-validation-policy: ## 🚀 Deploy the JWT validation policy
 
 # Propose a new idp
 propose-jwt-ms-validation-policy: ## 🚀 Propose the AAD as idp
+propose-jwt-ms-validation-policy: ## 🚀 Propose the AAD as idp. If AAD rolled keys, governance/jwt/set_jwt_ms_validation_policy_proposal.json will be updated and can be checked in
 	@echo -e "\e[34m$@\e[0m" || true
-	@./scripts/generate_jwt_proposal_payload.sh  --proposal-file "${KMS_WORKSPACE}"/proposals/set_jwt_ms_validation_policy_proposal.json
-	@CCF_PLATFORM=${CCF_PLATFORM} ./scripts/submit_proposal.sh --network-url "${KMS_URL}" --proposal-file "${KMS_WORKSPACE}"/proposals/set_jwt_ms_validation_policy_proposal.json --certificate_dir "${KEYS_DIR}" --member-count ${MEMBER_COUNT}
-
+	@./scripts/generate_jwt_proposal_payload.sh  --proposal-file "${PWD}"/governance/jwt/set_jwt_ms_validation_policy_proposal.json
```

Comment:
Using pwd could break the command when you are running make command with `-C` option from a different directory than usual. 

```bash
-C dir, --directory=dir
            Change  to  directory  dir before reading the makefiles or doing anything else.  If multiple -C options are specified, each is interpreted relative to the previ
ous one: -C / -C
            etc is equivalent to -C /etc.  This is typically used with recursive invocations of make.
```

It would be nicer if you can avoid it since you are making it production ready